### ISSUE TITLE
Logo / Icon

### ISSUE THREAD

**Issue by mikernet** [0xd4d/dnSpy#1535]
Hey,

Would you like a logo and app icon set designed for dnSpy? 

Cheers :)

**Comment by 0xd4d** [comment]
It already uses the VS 'assembly' image as the exe icon, which I think is good.

Where should the logo be shown?

If you want to create one, feel free to do. Open an issue with the logo, and I'll see if enough people like it or not (up/down votes).

**Comment by mikernet** [comment]
I thought you might want something customized and unique for dnSpy that would be incorporated into icon, i.e. how Visual Studio uses a minimalistic version of their logo for its icon and they both match. Right now dnSpy's icon is generic and I've seen it in other places, so I wanted to add a touch of flare to it for ya to set it apart a bit. Even if it's as simple as adding a bit of color to it if you want to stick with the overall look - right now it *feels* like an icon that comes packaged by default with windows. Looks a bit out of place in my taskbar :P haha

**Comment by 0xd4d** [comment]
Yeah, if people like it, we can change the app icon too. :) I'll just let people vote by using the thumbs up/down emojis when you've created the icons.

**Comment by greenozon** [comment]
@0xd4d  whats the threshold for accepting this feature? :)
